status-payloads-spec.md

```diff
@@ -162,5 +235,4 @@ TBD.
 
 ### Why are you using Transit and Protobuf?
 
-Transit was initially chose for encoding, and Protobuf was added afterwards. This is partly due to the history of the protocol living inside of `status-react`, which is written in Clojurescript. In future versions of payload and data sync client specifications it is likely we'll move towards Protobuf only. See e.g. [Dasy](https://github.com/vacp2p/dasy) for a research proof of concept.
-<!-- I would remove the link to dasy, I find it a bit confusing, and the repo just implements something totally different and a fraction of the functionalities -->
+Transit was initially chose for encoding, and Protobuf was added afterwards. This is partly due to the history of the protocol living inside of `status-react`, which is written in Clojurescript.
```

Comment:
do we have consensus on the future of this?  Are we planning on removing transit?

Reply:
We have consensus that we want to remove transit, but given that if that happens needs to happen before code freeze, not sure we will get around, so I would assume that it does not happen unless it does :)

Reply:
keep them both, just curious.  I'm definitely on the plan of mapping out exactly how v1 works, then we can discuss how to move forward.